libs/langchain/langchain/vectorstores/elastic_vector_search.py

```diff
@@ -51,9 +50,19 @@ def _default_script_query(query_vector: List[float], filter: Optional[dict]) ->
     }
 
 
-@deprecated("0.0.265", alternative="ElasticsearchStore class.", pending=True)
 class ElasticVectorSearch(VectorStore):
```

Comment:
Should we keep the deprecation notice? It sends a warning to users letting them know that there's a better implementation. It doesn't necessarily mean that this implementation will be removed soon.

Reply:
Hey @eyurtsev thanks for review. Will the notice "deprecated" be still mentioned in the [docs](https://integrations.langchain.com/vectorstores) if it continues to be there? Ideally we dont want this message there as gives the wrong signal. Thanks!